api-reference/upload-file-api/client-side-file-upload.md

```diff
@@ -225,55 +132,51 @@ echo("Auth params : " . json_encode($authenticationParameters));
 {% endtab %}
 {% endtabs %}
 
-{% hint style="danger" %}
-**Never publish your private key on client-side**\
-The Private API key should be kept confidential and only stored on your own servers.
-{% endhint %}
+## Client-side file upload examples
 
-## Examples
+The example below demonstrates only basic usage. Refer to [these examples](server-side-file-upload.md#examples) in the server-side upload section to learn about different use cases. The only difference between client-side and server-side uploads is how API authentication works.
 
-The example below demonstrates only basic usage. Refer to [these examples](server-side-file-upload.md#examples) in the server-side upload section to learn about different use-cases. The only difference between client-side and server-side upload is how API authentication works.
+Make sure you have implemented the secure API in the backend that can return the `signature`, one-time `token`, and `expire` parameters.
 
-Make sure you have implemented `authenticationEndpoint` endpoint on your server as shown [here](client-side-file-upload.md#how-to-implement-authenticationendpoint-endpoint) before using the below examples.
+Best way is to follow [quick start guides](../../getting-started/quickstart-guides/README.md) for your programming lanaguge.
 
 {% tabs %}
 {% tab title="JavaScipt SDK" %}
 {% code title="index.html" %}
 ```markup
 <form action="#" onsubmit="upload()">
-	<input type="file" id="file1" />
-	<input type="submit" />
+  <input type = "file" id="file1" />
+  <input type = "submit" />
 </form>
 <script type="text/javascript" src="../dist/imagekit.js"></script>
 
 <script>
-    /* 
-        SDK initilization
-        
-        authenticationEndpoint should be implemented on your server 
-        as shown above 
-    */
-    var imagekit = new ImageKit({
-        publicKey : "your_public_api_key",
-        urlEndpoint : "https://ik.imagekit.io/your_imagekit_id",
-        authenticationEndpoint : "https://www.yourserver.com/auth"
-    });
-    
-    // Upload function internally uses the ImageKit.io javascript SDK
-    function upload(data) {
-        var file = document.getElementById("file1");
-        imagekit.upload({
-            file : file.files[0],
-            fileName : "abc.jpg",
-            tags : ["tag1"]
-        }, function(err, result) {
-            console.log(arguments);
-            console.log(imagekit.url({
-                src: result.url,
-                transformation : [{ height: 300, width: 400}]
-            }));
-        })
-    }
+  /*  
+    SDK initialization
+  */
+  var imagekit = new ImageKit({
+    publicKey: "your_public_api_key",
+    urlEndpoint: "https://ik.imagekit.io/your_imagekit_id",
+  });
+
+  // Upload function internally uses the ImageKit.io javascript SDK
```

Comment:
Why are we not adding authentication request example in this like we have for jquery?

Reply:
as disscussed on a call to not change this